### PR TITLE
[WIP] Adding support for FMUv5X

### PR DESCRIPTION
**Pull request by davids5**
TODOS:

- [ ] Drivers
  - [ ]  Digital power module 
    - [ ]    Add INA226
    - [ ]   Support
- [ ]  BMI088
- [ ]  ISM330
- [ ]  BMP388
- [ ]  A71CH

FYI @dagar @LorenzMeier @eyeam3

### CODE REVIEW

File: boards/px4/fmu-v5x/firmware.prototype [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+    "board_id": 51,
+    "magic": "PX4FWv1",
+    "description": "Firmware for the PX4FMUv5C board",
```
Comment (dagar) [inline boards/px4/fmu-v5x/firmware.prototype]:
```suggestion
    "description": "Firmware for the PX4FMUv5x board",
```

Comment (davids5) [reply boards/px4/fmu-v5x/firmware.prototype]: done, but Cap X

**Comment by dagar** [comment]
Rebased on master with a few minor changes squashed into the first commit to sync up with fmu-v5 changes since the PR was opened. Things like the updated CONFIG_MAX_TASKS, default modules (dmesg, optical_flow, etc). I also synced up the variants (fixedwing, multicopter, rover, rtps, stackcheck) with default.

I'll merge to master once this passes CI.

**Comment by dagar** [comment]
CI found more churn (board dma allocate includes). Let's try to merge these things early.

**Pull request merged**